hosting/locale/de/LC_MESSAGES/django.po

```diff
@@ -482,6 +482,12 @@ msgid ""
 "contact Data Center Light Support."
 msgstr ""
 
+#~ msgid "Ipv4"
```

Comment:
yes, removed both IPv4 and IPv6 intentionally, since they were abbreviations without translations.